README.md

```diff
@@ -809,6 +810,38 @@ The following props are part of each element in `items`:
 - `onReorderClick: (index, newIndex) => (event) => void`: Returns a function that swaps the items at `index` with `newIndex`.
 - `readonly`: A boolean value stating if the array item is readonly.
 
+### Error List template
+
+To take control over how the form errors are displayed, you can define an *error list template* for your form.
```

Comment:
I'd precise we're speaking of the error list at the top of the form, not the individual field ones.